src/BackupStack.ts

```diff
@@ -45,6 +55,36 @@ export class BackupStack extends Stack {
 
   }
 
+  setupKmsKeyForBackupBuckets(props: BackupStackProps) {
+    const key = new kms.Key(this, 'backup-sse-key', {
+      description: 'Key for S3 backup buckets SSE',
+      alias: Statics.aliasBackupKmsKey,
+    });
+
+    key.addToResourcePolicy(
+      new iam.PolicyStatement({
+        effect: iam.Effect.ALLOW,
+        actions: ['kms:Encrypt', 'kms:Decrypt', 'kms:ReEncrypt*', 'kms:GenerateDataKey*'],
+        resources: ['*'],
```

Comment:
Moet dit niet gedefinieerd worden voor specifieke buckets?

Reply:
Dit is specifiek voor deze bucket omdat het om een resource policy gaat (`addToResourcePolicy`)

Reply:
Dacht dat het dan nog steeds over meerder resources ging, maar bij een specifieke resource policy dus blijkbaar niet